### PR TITLE
feat(git): add push option for selected bookmarks in multi-select

### PR DESCRIPTION
**Pull request by neil-sriv**
## Summary

- When multiple revisions with bookmarks are selected and the git menu is opened (`g`), add a new push option that runs `jj git push --remote <remote> --bookmark <name> ...` for only the bookmarks on the selected revisions
- Deduplicate bookmark names across selected revisions and skip conflicted bookmarks
- Bind the new option to shortcut `b` in the push category

## Test plan

- [x] `go test ./internal/ui/git/`
- [x] Select multiple revisions with bookmarks, press `g`, filter push options, and confirm the aggregated bookmark push item appears
- [x] Run the new option and verify only the selected bookmarks are pushed to the chosen remote

### CODE REVIEW

File: internal/ui/git/git.go [hunk 1]
```diff
@@ -677,6 +683,22 @@ func (m *Model) createMenuItems() []item {
 				command:  jj.GitPush(flags...),
 				key:      "c",
 			})
+
+		if len(selectedBookmarks) > 0 {
+			var bookmarkFlags []string
+			for _, name := range selectedBookmarks {
+				bookmarkFlags = append(bookmarkFlags, "--bookmark", name)
+			}
+			flags := append([]string{"--remote", selectedRemote}, bookmarkFlags...)
```
Comment (idursun) [inline internal/ui/git/git.go:693]: If the selected bookmarks have different remotes, then this command would push all of them to the currently selected bookmark. This needs to be generated for the bookmarks that match the current selected bookmark.

File: internal/ui/git/git_test.go [hunk 2]
```diff
@@ -91,6 +91,27 @@ func Test_PushChange(t *testing.T) {
...
+	commandRunner.Expect(jj.GitPush("--remote", "", "--bookmark", "feature-a", "--bookmark", "feature-b"))
+	defer commandRunner.Verify()
+
+	selected := jj.NewSelectedRevisions(&jj.Commit{ChangeId: changeId1}, &jj.Commit{ChangeId: changeId2})
+	op := NewModel(test.NewTestContext(commandRunner), selected)
+	test.SimulateModel(op, op.Init())
+	_ = test.RenderImmediate(op, 100, 40)
+
+	test.SimulateModel(op, func() tea.Msg { return intents.GitOpenFilter{} })
+	test.SimulateModel(op, test.Type("git push --bookmark feature-a --bookmark feature-b"))
```
Comment (idursun) [inline internal/ui/git/git_test.go]: I don't think this usage is correct here.

The following is a better usage.

```go
	test.SimulateModel(op, intents.Invoke(intents.GitFilter{Kind: intents.GitFilterPush}))
	test.SimulateModel(op, intents.Invoke(intents.GitApplyShortcut{Key: "b"}))
```

File: internal/ui/git/git.go [hunk 3]
```diff
@@ -631,12 +631,18 @@ func (m *Model) createMenuItems() []item {
...
 	for _, commit := range revisions.Revisions {
 		bookmarks := loadBookmarks(m.context, commit.GetChangeId())
 		for _, b := range bookmarks {
 			if b.Conflict {
 				continue
 			}
+			if !seenBookmarks[b.Name] {
+				seenBookmarks[b.Name] = true
+				selectedBookmarks = append(selectedBookmarks, b.Name)
+			}
```
Comment (idursun) [inline internal/ui/git/git.go]: `BookmarkList(... -a -r ...)` can return remote-only bookmarks (`b.Local == nil`), but `jj git push --bookmark <name>` expects a local bookmark name. Should this skip bookmarks where `b.Local == nil` before adding them to `selectedBookmarks`?

**Review by idursun — CHANGES_REQUESTED** [review]
Even though this is in draft I thought I could leave a quick review.

**Comment by neil-sriv** [comment]
Appreciate the review! Had left this in draft till I got a response on https://github.com/idursun/jjui/issues/701 but happy to move forward with it.

**Review by idursun — APPROVED** [review]
Hey, thanks for addressing the review feedback. The remote filtering, remote-only bookmark handling, and test interaction changes all look good.

One remaining edge case: `HasRemote` currently ignores the remote bookmark’s `Tracked` field. An untracked bookmark on the selected remote can therefore be included in the batch, causing `jj git push` to reject the entire command. I'll fix that separately after merging.
